app/Http/Controllers/API/CustomTourController.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Requests\API\CustomTourRequest;
+use App\Models\CustomTour;
+use Illuminate\Http\Request;
+
+class CustomTourController extends BaseController
+{
+    public function store(CustomTourRequest $request)
+    {
+        $validated = $request->validated();
+
+        // Perform basic data sanitization using strip_tags
+        $sanitizedData = $this->sanitizeData($validated);
+
+        $record = CustomTour::create(['tour_json' => json_encode($sanitizedData)]);
+
+        return response()->json(['message' => 'Custom tour created successfully!', 'custom_tour' => $record], 201);
+    }
+
+    public function show(Request $request, $id)
+    {
+        $custom_tour = CustomTour::find($id);
+
+        if (!$custom_tour) {
+            return response()->json(['message' => 'Custom tour not found'], 404);
+        }
```

Comment:
Using [implicit route-model binding](https://laravel.com/docs/10.x/routing#implicit-binding) can obviate lines lines 25-29.

For example, defining the route like:
```php
Route::get('/custom-tours/{customTour}', [CustomTourController::class, 'show']);
```
... will allow you to write the method signature like:
```php
public function show(Request $request, CustomTour $customTour)
```

This will retrieve the `CustomTour` with the provided `id` or throw a 404 if it isn't found.

Reply:
Thanks Zach, I'll implement this change as part of this Pivotal story: https://www.pivotaltracker.com/n/projects/2666621/stories/186213538